tutorials/k8s_workload.md

```diff
@@ -0,0 +1,352 @@
+# Tutorial: Protecting Containerized Assets with OSC on Kubernetes
+
+Given an existing [Kubernetes compatible environment](/gettingstarted/requirements.md#user-content-kubernetes) and a [deployed instance of OSC](/gettingstarted/installing_ost.md), this tutorial provides the necessary steps for setting up OSC to protect a single workload instance in Kubernetes. 
+
+These steps include: 
+* Deployment of a security appliance instance.
+* Definition of the workload under protection.
+* Redirection of the traffic to be protected through the security appliance.
+
+## Setup Requirements
+
+### Virtualization Environment  
+
+In Kubernetes, create two pods:
+1. The **attacker** pod:  
+This pod will have the container from which we will *ping* the victim pod. 
+Create the pod configuration file:
+```sh
+$ cat >~/attacker-busybox.yaml <<EOL
+apiVersion: v1
+kind: Pod
+metadata:
+  name: attacker-busybox
+  namespace: default
+spec:
+  containers:
+  - image: busybox
+    command:
+      - sleep
+      - "3600"
+    imagePullPolicy: IfNotPresent
+    name: attacker-busybox
+  restartPolicy: Always
+EOL
+```
+Create the pod:
+```sh
+$ kubectl create -f ~/attacker-busybox.yaml
+```
+2. The **protected** pod:  
+The protected pod that will be pinged by the attacker. Observe it has a label which will be used later for creating a security group member in OSC.  
+Create the pod configuration file:  
+```sh
+$ cat >~/victim-busybox.yaml <<EOL
+apiVersion: v1
+kind: Pod
+metadata:
+  name: victim-busybox
+  namespace: default
+  labels:
+    env: production
```

Comment:
can we highlight this line?

Reply:
Not on code snippets AFAIK, but I will add a more explicit note above.